packages/frontend/src/view/pages/user/UserPage.tsx

```diff
@@ -55,13 +56,15 @@ function UserPage(props: UserPageProps) {
     >
       <ContentWrapper className="flex flex-col gap-12">
         <UserProfile
-          starkKey={props.starkKey}
           ethereumAddress={props.ethereumAddress}
+          myOwnProfile={myUserPage}
```

Comment:
Maybe `myUserPage` and `myOwnProfile` could be renamed to simply `isMine` or `isOwner`. Regardless there should be one name used across all components

Reply:
I like `isMine`, done